### PR TITLE
new course on `DeepLearning.ai`

### PR DESCRIPTION
**Pull request by leo-gan**
Added a new course on [DeepLearning.ai](https://learn.deeplearning.ai/functions-tools-agents-langchain)
Added the LangChain `Wikipedia` link. Probably, it can be placed in the "More" menu.

### CODE REVIEW

File: docs/docs/additional_resources/tutorials.mdx [hunk 1]
```diff
@@ -1,15 +1,19 @@
 # Tutorials
 
-Below are links to tutorials and courses on LangChain. For written guides on common use cases for LangChain, check out the [use cases guides](/docs/use_cases).
+Below are links to tutorials and courses on [LangChain](https://en.wikipedia.org/wiki/LangChain).
```
Comment (baskaryan) [inline docs/docs/additional_resources/tutorials.mdx:3]: feel like we probably don't need to link to wikipedia from our own docs? if wikipedia is better at describing us than our docs than 😅

Comment (leo-gan) [reply docs/docs/additional_resources/tutorials.mdx:3]: As you wish. I thought the new Wikipedia link is great.

**Comment by leo-gan** [comment]
@baskaryan Any comments?

**Comment by leo-gan** [comment]
@efriis Could you, please, review it?

**Review by efriis — APPROVED** [review]
Thanks @leo-gan

**Comment by leo-gan** [comment]
@efriis Thank you!

**Pull request merged**